compiler/src/formatting/fmt.re

```diff
@@ -260,6 +260,94 @@ let needs_grouping = (~parent, ~side: infix_side, expr) => {
   };
 };
 
+let get_op_and_assignment = new_value => {
```

Comment:
This name feels confusing to me out of context. I want a name that communicates that it gets the operator and assignment value from an infix assignment expression, but obviously that seems unnecessarily wordy with a name like `get_infix_assignment_op_and_value`.

Reply:
What about `get_infix_parts`?